packages/entity/src/EntityDatabaseAdapter.ts

```diff
@@ -69,12 +69,14 @@ export interface QuerySelectionModifiers<TFields> {
 }
 
 export interface TableQuerySelectionModifiers {
-  orderBy?: {
-    columnName: string;
-    order: OrderByOrdering;
-  }[];
-  offset?: number;
-  limit?: number;
+  orderBy:
+    | {
+        columnName: string;
+        order: OrderByOrdering;
+      }[]
+    | undefined;
+  offset: number | undefined;
+  limit: number | undefined;
```

Comment:
For APIs it's usually nicer to be able to omit a field than to need to write `{ unusedOption: undefined }`. Depending on how this code is called into, it might be better to write: `{ orderBy?: {...} | undefined }` to allow both.